compiler/src/main/scala/aqua/compiler/AquaCompiler.scala

```diff
@@ -58,7 +59,10 @@ class AquaCompiler[F[_]: Monad, E, I: Order, S[_]: Comonad, C: Monoid: Picker](
         // Lift resolution to CompileRes
         modules <- resolution.toEitherT[CompileWarns]
         // Generate transpilation functions for each module
-        transpiled = modules.map(body => transpile(body))
+        transpiled = modules.map { m =>
+          val importIds = m.imports.view.mapValues(_.show).toMap
+          m.copy(body = transpile(m.body).map(_.map(_.setImportPaths(importIds))))
```

Comment:
nit: Maybe pass imports to `transpile` and set them there instead of `map(_.map(...))`